.github/workflows/release.yml

```diff
@@ -46,8 +50,6 @@ jobs:
           role-duration-seconds: 3600
           role-session-name: ${{ env.PROVIDER }}@githubActions
           role-to-assume: ${{ secrets.AWS_CI_ROLE_ARN }}
-      - name: Unshallow clone for tags
```

Comment:
In some CIs that's part of computing the version and can trip up versioning.

Reply:
I just moved it up sooner.